readme.md

```diff
@@ -170,19 +176,37 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Question:** You may notice that you can add custom error messages to individual fields failing. Try adding these and take a look at the error message received. How can you make use of those specific messages for each field?
 
-* **Your Answer:**
+* **Your Answer:** With `async/await` we can add a `try/catch` block and then `console.log()` any errors we might get back. We can then use the console to help debug.
 
 ---
 
 - [ ] With Mongo, it is simple to create complex data structures. Add a `characters` field to your Series model that is an array of objects. Each object should have a `name` and `image_url`. Only the `name` field on the character should be required. _Note: Don't forget to change your select statements to include the `characters` field!_
 
 * **Question:** Take a look at the response from making a new series with a character. What field was created that you did not have to define?
 
-* **Your Answer:**
+* **Your Answer:** Mongoose created a unique key identifies in the `characters` array.
 
 * **Question:** With the current routes that we have, how would you upate the name of a character in a series?
 
-* **Your Answer:**
+* **Your Answer:** I would need to update the `PUT` route and attempted something like the following:
+
+```js
+router.put("/:id/:id", async (req, res, next) => {
+  const status = 200;
+  const response = await Series.findOneAndUpdate(
+    { _id: req.params.id },
+    { title: req.body.title },
+    { start_year: req.body.start_year },
+    { season_count: req.body.season_count },
+    { characters: req.body.characters.name },
+    { new: true }
+  );
+
+  res.json({ status, response });
+});
+```
+
+I was trying to see if I could drill down in the id's to get to the character name, but that doesn't work here. Hoping we can cover the solution with more detail in class.
```

Comment:
Let me know if you're still having trouble with this. You should just be able to reconstruct the model from scratch and/or just update the `characters` key. You will need to replace the entire `characters` key with a new representation of all characters.